v4/source/teams.yaml

```
@@ -39,6 +39,19 @@
           $ref: '#/responses/Unauthorized'
         '403':
           $ref: '#/responses/Forbidden'
+      x-code-samples:
+        - lang: 'Go'
+          source: |
+            import "github.com/mattermost/platform/model"
+
+            Client := model.NewAPIv4Client("https://your-mattermost-url.com")
+            Client.Login("email@domain.com", "Password1")
+
+            newTeam, err := Client.CreateTeam(&model.Team{
+              Name: <TEAM_NAME>,
```

I see this code example as runnable example, not as a temple. Hence, I would rather fill this request with some arbitrary data.
```suggestion
              Name: "someTeamName",
```

---

@hanzei It sounds good, I can modify the four PRs at once and add a comment on this one when done

---

Would you mind just starting with this one? Maybe other reviewer have other objects. I want to avoid that you have to do too much additional work.

---

You are right, I will start with this one

---

@hanzei I replaced all `<XXX>` with some arbitrary data